__tests__/app.test.js

```diff
@@ -197,3 +198,68 @@ describe('POST/api/articles/:article_id/comments', () => {
 			});
 	});
 });
+
+describe('PATCH/api/articles/:article', () => {
+	test('responds with a 200 status and the article object with updated vote count', () => {
+		return request(app)
+			.put('/api/articles/1')
+			.send({ inc_votes: 10 })
+			.expect(200)
+			.then(({ body }) => {
+				expect(body).toEqual(
+					expect.objectContaining({
```

Comment:
This test would be more secure if you checked the exact article you're patching - at the very least you'd want to make sure the article_id is correct 😄